### ISSUE TITLE
Getting black background for images after using saveImage

### ISSUE THREAD

**Issue by Marchiuzzz** [spatie/pdf-to-image#165]
Hi, I can see there was an issue report regarding this problem, which seems it should've been solved by now (since they were posted a few years ago), however I'm still facing it. This doesn't happen to all file though. Are there any solutions for this? I'm using version 1.8.2
![image](https://user-images.githubusercontent.com/73941456/103360638-a908c580-4ac1-11eb-958d-268f176d1dfb.png)

Here's my code:
```
$pdf = new \Spatie\PdfToImage\Pdf($filepath);
for ($i = 1; $i <= $pdf->getNumberOfPages(); $i++) {
   $pdf->setPage($i)->setOutputFormat('png')->saveImage($imagePath);
}
```

**Comment by kadircanerergun** [comment]
Same problem for me? 

Version: ImageMagick 6.9.10-68 Q16 x86_64 2020-12-15 

Centos 7
PHP 7.4

**Comment by Marchiuzzz** [comment]
@kadircanerergun I couldn't find a solution but it seems it is a problem with either GhostScript or imageMagick. My solution was to run shell commands using ImageMagick's "convert". I don't really like this solution but it's the only one that worked for me.

**Comment by Marchiuzzz** [comment]
I have figured this is an issue when pdfs contain CMYK colorspace items (vectors, text, etc.). Adding setColorspace(\Imagick::COLORSPACE_RGB) actually preserves the vector colors but still issue with black background persists. I figured that running image magick's covert command with "-alpha flatten" parameter results in a proper image, is there a way to set alpha using this package? Did this ever get implemented? https://github.com/spatie/pdf-to-image/pull/155

![image](https://user-images.githubusercontent.com/73941456/111955488-3b819980-8af2-11eb-9378-3e4fb869a38b.png)
